Heart/Message/Application/NewMessage.php

```diff
@@ -22,7 +22,7 @@ public function __construct(
     ) {
     }
 
-    public function handle(array $payload): void
+    public function persist(array $payload): void
     {
         $messageDTO = NewMessageDTO::make($payload);
```

Comment:
What if the payload is invalid and the deserialization fails? I think we should be dealing with exceptions here, right? The same goes to storing the message into the database.